server/chat-plugins/datasearch.ts

```diff
@@ -1988,6 +2124,16 @@ function runItemsearch(target: string, cmd: string, canAll: boolean, message: st
 		target = target.substr(0, lastCommaIndex);
 	}
 
+	const sanitized_targets = [];
+	for (const index of target_split.keys()) {
+		const local_target = target_split[index].trim();
+		if (local_target.startsWith('random') && cmd === 'randitem') {
+			randomOutput = parseInt(local_target.substr(6));
+		} else {
+			sanitized_targets.push(local_target);
+		}
+	}
+	target = sanitized_targets.join(',');
```

Comment:
```suggestion
	const sanitizedTargets = [];
	for (const index of targetSplit.keys()) {
		const localTarget = targetSplit[index].trim();
		if (localTarget.startsWith('random') && cmd === 'randitem') {
			randomOutput = parseInt(localTarget.substr(6));
		} else {
			sanitizedTargets.push(localTarget);
		}
	}
	target = sanitizedTargets.join(',');
```
Our style convention is to use `camelCase` for variable names.